scripts/generate.sh

```diff
@@ -1,15 +1,24 @@
 # generates Node Mapnik documentation using documentationjs
```

Comment:
@mapsam  - I noticed this script does not have `set -eu` at the top- can it (to ensure it stops on error)?